### PR TITLE
docs:  ref `<Image>` in docs ui components

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Comment by OsamaAbdellateef** [comment]
I don't think that this needs a changeset right ?! @sarahgm

File: themes/theme-docs/src/components/Dialog.styles.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { ThemeComponent, cva } from '@marigold/system';
 
 export const Dialog: ThemeComponent<'Dialog'> = {
-  closeButton: cva('absolute right-2 top-2', {
+  closeButton: cva('', {
```
Comment (sarahgm) [inline themes/theme-docs/src/components/Dialog.styles.ts:4]: I think you have to add this again :)

**Review by sarahgm — CHANGES_REQUESTED** [review]

File: themes/theme-docs/src/components/Dialog.styles.ts (hunk 1 above)
Comment (OsamaAbdellateef) [reply themes/theme-docs/src/components/Dialog.styles.ts:4]: I added it directly on CloseButton component @sarahgm

Comment (sarahgm) [reply themes/theme-docs/src/components/Dialog.styles.ts:4]: Bildschirmfoto 2024-10-21 um 13 33 24
but this doesn't look right to me? :O

**Review by sarahgm — CHANGES_REQUESTED** [review]

File: themes/theme-docs/src/components/Dialog.styles.ts (hunk 1 above)
Comment (sarahgm) [reply themes/theme-docs/src/components/Dialog.styles.ts:4]: I think we said the button should align with the headline

**Review by sarahgm — APPROVED** [review]

**Pull request merged**